docs/source/usage/index.rst

```diff
@@ -225,6 +225,58 @@ In order to use an external provisioning service, simply attach the node to the
 | Attach Network to Node        | ``openstack esi node network attach (--network <network> | --port <port>) <node>`` |
 +-------------------------------+------------------------------------------------------------------------------------+
 
+Serial Console
+---------------
+
+Ironic nodes can be accessed through configuring a web or serial interface.
+This documentation was taken from the Ironic Xena release.
+
+In order to access a node using a serial console, the user must configure the node's console interface. 
```

Comment:
Under our model, it's the admin or owner that configures the node's console interface; it's not something a lessee should be able to do. This is a place where I'd say it's appropriate to say something like "the owner or lessee must configure the node's console interface as described in some-link".

Then you can get rid of the 'For example' line.